<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VerCarro</class>
 <widget class="QWidget" name="VerCarro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ver Carro</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icon.ico</normaloff>../icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
	background-color:white</string>
  </property>
  <widget class="QPushButton" name="btnDefinicoes">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>60</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgb(255, 245, 166);
	border:none;
	color:rgb(95, 91, 61);
	border -bottom: 5px solid rgb(162, 155, 105)
}
QPushButton:hover{
	background-color:rgb(229, 217, 150);
	border-bottom: 5px solid rgb(162, 155, 105)
}
QPushButton:pressed {
	background-color:rgb(255, 242, 165);
	border-bottom: 5px solid rgb(162, 155, 105)
}</string>
   </property>
   <property name="text">
    <string>Definições</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEstatistica">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>140</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color:rgb(255, 245, 166);
	border:none;
	color:rgb(95, 91, 61);
	border -bottom: 5px solid rgb(162, 155, 105)
}
QPushButton:hover{
	background-color:rgb(229, 217, 150);
	border-bottom: 5px solid rgb(162, 155, 105)
}
QPushButton:pressed {
	background-color:rgb(255, 242, 165);
	border-bottom: 5px solid rgb(162, 155, 105)
}</string>
   </property>
   <property name="text">
    <string>Estatísticas</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Carro:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Tipo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Observações:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNomeCarro">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Carro XPTO</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTipoCarro">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Eletrico</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>431</width>
     <height>131</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>429</width>
      <height>129</height>
     </rect>
    </property>
    <widget class="QLabel" name="labelObsCarro">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>411</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ano Xptodsajbjdkbk,a,bd</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QCommandLinkButton" name="commandButtonVoltar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QCommandLinkButton
{
	border: none;
}

QCommandLinkButton:hover
{
	border: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../setavoltar.ico</normaloff>../setavoltar.ico</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
